server/src/Infrastructure/Persistence/ApplicationDbContext.cs

```diff
@@ -14,9 +14,9 @@ public class ApplicationDbContext : DbContext
     public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
     {
     }
-
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
         modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
```

Comment:
I think we no longer need configurations. Foreign keys and owned entities can be configured using data annotations